charts/linkerd2/templates/web-rbac.yaml

```diff
@@ -25,6 +25,19 @@ rules:
   verbs: ["list"]
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: linkerd-web-check-injector
+  namespace: default
+  labels:
+    {{.ControllerComponentLabel}}: web
+    {{.ControllerNamespaceLabel}}: {{.Namespace}}
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["create"]
```

Comment:
could this rule just be added to the `linkerd-web-check` role rather than adding a new role?

Reply:
The role that rule belongs to is attached to the `default` namespace whereas `linkerd-web-check` role is attached to the namespace where Linkerd is installed. Please note that `linkerd-web-check` role is gone after the rebase I've just done from master. Those permissions were nicely merged into `linkerd-web` role in #3694.

I used the default namespace instead the one Linkerd is installed to because the latter had the annotation `linkerd.io/inject: disabled`, so the pod created during the check was never injected.